### ISSUE TITLE
Bug: MultiMesh uses_colors error vs use_colors property

### ISSUE THREAD

**Issue by RobertBColton** [godotengine/godot#100988]
### Tested versions

4.3 dev

### System information

Godot v4.3.stable - Windows 10.0.22631 - GLES3 (Compatibility) - NVIDIA GeForce RTX 4050 Laptop GPU (NVIDIA; 32.0.15.6070) - Intel(R) Core(TM) Ultra 7 155H (22 Threads)

### Issue description

Keep getting an error message using colors because its looking for the wrong property.
Following this tutorial:
https://docs.godotengine.org/en/stable/classes/class_multimesh.html#class-multimesh-property-use-colors

```
E 0:00:04:0397   game.tscn::GDScript_a3sjo:43 @ _ready(): Condition "!multimesh->uses_colors" is true.
  <C++ Source>   drivers/gles3/storage/mesh_storage.cpp:1714 @ multimesh_instance_set_color()
  <Stack Trace>  game.tscn::GDScript_a3sjo:43 @ _ready()
```

### Steps to reproduce

Create multimesh and enable `use_colors` and then set instance colors.

### Minimal reproduction project (MRP)

Official tutorial does it if you just switch on `use_colors`
https://docs.godotengine.org/en/stable/classes/class_multimesh.html#class-multimesh-property-use-colors

**Comment by RobertBColton** [comment]
Sorry, this one was a false alarm.

The manual clearly states that you need to set it before instance_count:
`This means that you need to call this method before setting the instance count, or temporarily reset it to 0.`

That made the error go away, I'm not sure if this could be made clearer in docs though.

**Comment by clayjohn** [comment]
Thank you for following up after your initial report! I have made a PR that improves the error report so that other users don't get bit by the same issue.

**Comment by RobertBColton** [comment]
Yeah, so I am making a city builder in Godot. I plan to release the source to the good people of Godot.

![Image](https://github.com/user-attachments/assets/0d133852-1fb2-4691-beae-ac4febc48655)
